src/views/projects/Patch.svelte

```diff
@@ -159,6 +160,17 @@
     currentRevision.base,
     currentRevision.oid,
   );
+
+  const badgeColor: Variant =
```

Comment:
I always find nested ternary conditionals hard to read. Could we convert it to if/else?

Reply:
Yes that should work too

Reply:
9111fdd